### DISCUSSION TITLE
How to get formData in vercel-edge/workers

### DISCUSSION THREAD

**Discussion by cosbgn** [nitrojs/nitro#1718] [General]
So far I tried:

```js

export default defineEventHandler(async (event) => {
    const form = await readFormData(event)
    // I've also tested with
    // const form = await toWebRequest(event).formData()
    return "DONE"
})

```

Both work locally but hung and then die with a 502 on vercel-edge. Any ideas?

**Comment by Hebilicious** [comment]
@cosbgn this could be a bug, could you please open an issue ?

**Reply by cosbgn** [reply]
@Hebilicious  I did on H3 - https://github.com/unjs/h3/issues/538

reproduction on https://github.com/cosbgn/edgenu/blob/main/server/api/form-test.post.js (edgenu.vercel.app)

**Comment by pi0** [comment]
Let's track via https://github.com/unjs/nitro/issues/1721
